restapi.go

```diff
@@ -2874,6 +2874,76 @@ func (s *Session) FollowupMessageDelete(appID string, interaction *Interaction,
 	return s.WebhookMessageDelete(appID, interaction.Token, messageID)
 }
 
+// ------------------------------------------------------------------------------------------------
+// Functions specific to stage instances
+// ------------------------------------------------------------------------------------------------
+
+// StageInstanceData is provided to StageInstanceCreate
+type StageInstanceData struct {
+	// ChannelID represents the id of the Stage channel
+	ChannelID string `json:"channel_id"`
+	// Topic of the Stage instance (1-120 characters)
+	Topic string `json:"topic"`
+	// PrivacyLevel of the Stage instance (default GUILD_ONLY)
+	PrivacyLevel *StageInstancePrivacyLevel `json:"privacy_level,omitempty"`
+	// SendStartNotification will notify @everyone that a Stage instance has started
+	SendStartNotification *bool `json:"send_start_notification,omitempty"`
+}
+
```

Comment:
I think it would be better to move it to `structs.go`

Reply:
In this case, Maybe move all Data struct from restapi.go to structs.go ? I follow the existing.

Tell me your vision

Reply:
We're might to do that in the future. But that should be done in a separate PR, as it doesn't fit the stage instances functionality.